examples/ai-core/src/embed-many/openai-compatible-togetherai.ts

```diff
@@ -0,0 +1,25 @@
+import 'dotenv/config';
+import { createOpenAICompatible } from '@ai-sdk/openai-compatible';
+import { embedMany } from 'ai';
+
+async function main() {
+  const togetherai = createOpenAICompatible({
+    apiKeyEnvVarName: 'TOGETHER_AI_API_KEY',
```

Comment:
let's completely remove the concept of env variables for openai compatible. it's an unnecessary complexity in this special case, users can just pass the env property and we have less API surface

Reply:
(it it only there for convenience to begin with, and here convenience becomes complexity)

Reply:
Done.